vizro-core/src/vizro/static/css/tabs.css

```diff
@@ -0,0 +1,58 @@
+.tabs {
+  display: flex;
+  flex-direction: column;
+  gap: 20px;
+  height: 100%;
+}
+
+.tabs__list {
+  align-items: flex-start;
+  align-self: stretch;
+  border-bottom: 1px solid var(--border-subtle-alpha-01);
+  display: flex;
+  gap: 16px;
+  height: 32px;
+}
+
+.tabs__panel {
+  height: calc(100% - 60px);
+}
+
+.tab__content {
+  height: 100%;
+}
+
+.tab__title {
+  align-items: flex-start;
+  border-bottom: 1px solid transparent;
+  color: var(--text-secondary);
+  display: flex;
+  font-size: 16px;
+  height: 32px;
+  line-height: 20px;
+  margin: 0;
+  padding: 0;
+}
+
+.tab__title:hover {
+  background-color: transparent;
+  border-bottom: 1px solid var(--text-active);
+}
+
+.tab__title[data-active] {
+  border-bottom: 1px solid var(--text-active);
+  color: var(--text-active);
+}
+
+.tab__title[data-active]:hover {
+  background-color: transparent;
+  border-color: 1px solid var(--text-active);
+}
+
+.tab__title[data-active] .mantine-Tabs-tabLabel {
+  color: var(--text-active);
+}
+
+.tab__content > .page-component-container > .container__title {
+  display: none;
+}
```

Comment:
Is this the code that hides the container title when it is the highest level container in the components attribute of the Tabs model?

If yes, can you explain

Reply:
Yes, exactly - can recommend reading about CSS combinators [here](https://www.w3schools.com/css/css_combinators.asp). In general I am leveraging the child selector, so only the immediate child will be affected. Because the inner containers are not an immediate child, the titles for them are shown, while the title for the outermost container is hidden.

Reply:
Please could you record somewhere (I'd suggest a comment in `Container.build`) that the title is hidden in the context of a tab using CSS and note the alternatives we discussed:
* an argument in the `Container.build` method that tells you whether the container is being built inside `Tabs` and then sets `hidden` attribute for the heading or just doesn't supply the element at all
* logic in `Tabs.build` that sets `hidden` attribute for the heading or uses `del` to remove the heading
* new field in `Container` like `short_title` to allow tab label to be set independently
* any other options we discussed yesterday that I forgot...

I like the CSS solution for now but maybe we'll want to change in future. e.g. semantically `hidden=True` is better than CSS `display: none` here, which is a very minor disadvantage of our current solution.